src/lisp/kernel/init.lsp

```diff
@@ -855,6 +856,7 @@ the stage, the +application-name+ and the +bitcode-name+"
                  `(my-do-time #'(lambda () ,form))))
            t)
 (export 'my-time)
+)
```

Comment:
I think this should just be removed altogether. It isn't referenced anywhere else.